Mathlib/Data/MvPolynomial/Division.lean

```diff
@@ -117,6 +126,77 @@ theorem coeff_modMonomial_of_le {s' s : σ →₀ ℕ} (x : MvPolynomial σ R) (
   x.modOf_apply_of_exists_add _ _ <| exists_add_of_le h
 #align mv_polynomial.coeff_mod_monomial_of_le MvPolynomial.coeff_modMonomial_of_le
 
+lemma neg_modMonomial (s : σ →₀ ℕ) (x : MvPolynomial σ R') :
+    (-x) %ᵐᵒⁿᵒᵐⁱᵃˡ s = -(x %ᵐᵒⁿᵒᵐⁱᵃˡ s) :=
+  x.neg_modOf _
+
+lemma add_modMonomial (s : σ →₀ ℕ) (x y : MvPolynomial σ R) :
+    (x + y) %ᵐᵒⁿᵒᵐⁱᵃˡ s = (x %ᵐᵒⁿᵒᵐⁱᵃˡ s) + (y %ᵐᵒⁿᵒᵐⁱᵃˡ s) :=
+  x.add_modOf _ _
+
+lemma sub_modMonomial (s : σ →₀ ℕ) (x y : MvPolynomial σ R') :
```

Comment:
This lemma and `neg` should be the only ones that need `R'`; I think the rest will work with `R`.

Reply:
everything except `X_sub_C_not_isUnit` ($X_i - c$ is not a unit) has been been generalized

Reply:
How is one supposed to type ᵐᵒⁿᵒᵐⁱᵃˡ? copy-paste?